### PR TITLE
Add Favicon to Doc and Example Page

### PR DESCRIPTION
**Pull request by mamamia5x**
Simple addition to give the site a favicon, making it look more professional.

### CODE REVIEW

File: site/doc.js [hunk 1]
```diff
@@ -208,6 +208,7 @@ const page = t("html", {}, [
 		t("meta", { charset: "utf-8", }),
 		t("style", {}, www.style(gstyle)),
 		t("style", {}, www.style(style)),
+    t("link", { rel: "icon", href: "/pub/img/kaboom.png"}),
```
Comment (slmjkdbtl) [inline site/doc.js:211]: is it possible to use tabs here to comply with the overall style?

File: site/examples.js [hunk 2]
```diff
@@ -117,6 +117,7 @@ const page = t("html", {}, [
 		t("title", {}, "KaBoom Examples"),
 		t("meta", { charset: "utf-8", }),
 		t("style", {}, www.style(style)),
+    t("link", { rel: "icon", href: "/pub/img/kaboom.png"}),
```
Comment (slmjkdbtl) [inline site/examples.js:120]: ditto ^

**Review by slmjkdbtl — CHANGES_REQUESTED** [review]

**Review by mamamia5x — COMMENTED** [review]
Alright, they are indented now.

**Comment by slmjkdbtl** [comment]
nice, thx!

**Pull request merged**